virtualizarr/tests/test_codecs.py

```diff
@@ -1,161 +1,141 @@
-from unittest.mock import patch
-
 import numpy as np
 import pytest
-from numcodecs import Blosc, Delta
+from zarr.codecs import BytesCodec
+from zarr.core.codec_pipeline import BatchedCodecPipeline
+from zarr.registry import get_codec_class
 
-from virtualizarr import ChunkManifest, ManifestArray
-from virtualizarr.codecs import get_codecs
-from virtualizarr.tests import (
-    requires_zarr_python,
-    requires_zarr_python_v3,
-)
-from virtualizarr.zarr import Codec
+from virtualizarr.codecs import convert_to_codec_pipeline, get_codecs
 
+arrayarray_codec = {"name": "numcodecs.delta", "configuration": {"dtype": "<i8"}}
+arraybytes_codec = {"name": "bytes", "configuration": {"endian": "little"}}
+bytesbytes_codec = {
```

Comment:
Following convention for global variables

```suggestion
ARRAYARRAY_CODEC = {"name": "numcodecs.delta", "configuration": {"dtype": "<i8"}}
ARRAYBYTES_CODEC = {"name": "bytes", "configuration": {"endian": "little"}}
BYTESBYTES_CODEC = {
```

Reply:
https://github.com/zarr-developers/VirtualiZarr/pull/429/commits/97bc7cd26136e0b0cafb6fc8f120cd42f793465e